docs-2.0/9.about-license/2.license-management-suite/3.license-manager.md

```diff
@@ -0,0 +1,200 @@
+# License Manager
+
+License Manager (LM) is a service that runs on a server for you to manage licenses. You can use an LM client that communicates with the LM service to load [license keys](2.license-center.md) and view license information, including the license validity period and purchased nodes. By configuring the LM service in the NebulaGraph database and its associated software, the validity of the license can be verified to ensure the normal use of the NebulaGraph database and its associated software.
+
+This article introduces how to deploy and use an LM service, as well as how to configure it within the Nebula Graph database and its associated software.
+
+## Preparations
+
+- [You have purchased a Vesoft license](../3.purchase-license.md).
+- You have obtained the desired LM installation package.
+  
+  !!! note
+
+        LM installation packages are sent to you by email after you purchase a license.
+
+## Notes
+
+An LM is a single-process service. To ensure the reliability and continuity of the LM, it is recommended that you use systemd to manage the LM and set a restart policy for the LM.
+
+## Install and start LM
+
+An LM can be installed on Linux amd64 or arm64 systems. 
+
+### TAR 
+
+1. Unpack the LM TAR package.
+
+  ```bash
+  tar -zxvf <name.tar.gz> -C <path>
+  ```
+
+  - `<name.tar.gz>`: The name of the LM TAR package.
+  - `<path>`: The installation path for the unpacked LM. If the `-C` parameter is not specified, it defaults to the current directory.
+  
+2. In the LM installation directory, start LM.
+
+  ```bash
+  nohup ./nebula-license-manager &
```

Comment:
not use nohup command here, it's not a good practice in production. Here, we just info user to use the nebula-license-manager binary to start the serivce, as for how to do it, it depends on user himself, he can use systemd or other tools.